packages/base/src/msg/factory.rs

```diff
@@ -1,10 +1,12 @@
-use crate::state::{CodeIds, RemoteOpts};
+use crate::msg::token::DenomMetadata;
+use crate::state::factory::{CodeIds, RemoteOpts};
 use cosmwasm_schema::{cw_serde, QueryResponses};
 use cosmwasm_std::{CosmosMsg, Decimal, Uint128};
 use cw_ownable::cw_ownable_execute;
 use drop_macros::pausable;
-use drop_staking_base::msg::token::DenomMetadata;
 use neutron_sdk::bindings::msg::NeutronMsg;
+#[allow(unused_imports)]
+use std::collections::HashMap;
```

Comment:
To avoid this weird `#[allow]` declaration, I usually import stuff by its full path in all declarative macros